ui/.eslintrc.js

```diff
@@ -19,11 +19,13 @@ module.exports = {
     sourceType: 'module',
   },
   rules: {
+    'no-console': ['error', {allow: ['warn', 'error']}],
```

Comment:
```suggestion
    'no-console': 2,
```

This would be enough.